by-language/python-sqlalchemy/README.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight: console
+
+############################################################
+Connect to CrateDB and CrateDB Cloud using Python/SQLAlchemy
+############################################################
+
+
+*****
+About
+*****
+
+Example programs demonstrating CrateDB's SQLAlchemy adapter and dialect.
+
+
+*****
+Usage
+*****
+
```

Comment:
### Improve documentation II

In this case, within the **Usage** section, it should clearly be focused on "library use" details, like, how to exactly use the `insert_bulk` function with pandas and Dask.

```python
from crate.client.sqlalchemy.support import insert_bulk

# pandas
df.to_sql(name=self.table_name, con=engine, if_exists="append", index=False, chunksize=bulk_size, method=insert_bulk)

# Dask
ddf.to_sql("testdrive", uri=DBURI, index=False, if_exists="replace", chunksize=10_000, parallel=True, method=insert_bulk)
```

What is currently within the **Usage** section, should go into an **Examples** section instead.

Reply:
Fixed, see above.